origocli/data/boilerplate/pipeline/csv-to-parquet/pipeline.json

```diff
@@ -10,10 +10,10 @@
       "chunksize": 100000
     },
     "write_to_processed": {
-      "write_to_latest": "1",
       "output_stage": "processed"
     },
     "write_to_latest": {
+      "write_to_latest": "1",
```

Comment:
Could this be set to `true` instead of `"1"`?

Reply:
Ideally I would like this kind of "static" config to be set on the pipeline template instead of each pipeline instance. Let's discuss how we can change pipeline router and config to fix this :-)